spec/forms/work_form_spec.rb

```diff
@@ -6,15 +6,28 @@
 RSpec.describe WorkForm do
   subject(:form) { described_class.new(work) }
 
-  let(:work) { build(:work) }
+  let(:work) { create(:work) }
 
   describe 'populator on files' do
+    let!(:blob) do
+      ActiveStorage::Blob.create_after_upload!(
```

Comment:
I spent an hour trying to find a better way to deal with AS-based models in FactoryBot but gave up.